### DISCUSSION TITLE
求一键长草中加回反选

### DISCUSSION THREAD

**Discussion by LeeSpo** [MaaAssistantArknights/MaaAssistantArknights#1083] [General]
最新版软件中，一键长草中能否加回反选，有时候挺实用
设置好后无需额外再勾选

**Comment by MistEO** [comment]
我觉得反选太反直觉了，是不好的用户体验。

后面可以会做成一个小后门，需要的自行在 gui.json 中开启。但不会在默认界面中加入这个

**Comment by zzyyyl** [comment]
现在的反选加回来了，可以自己在 `设置 - 界面设置` 里修改
